hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -817,6 +817,14 @@
       ids will be allocated in a single batch.
     </description>
   </property>
+  <property>
+    <name>ozone.scm.get.container.max.retry</name>
+    <value>4096</value>
+    <tag>OZONE, SCM</tag>
+    <description>
+      This property determines how many times SCM is going to retry getContainer operation for Ratis type container.
+    </description>
+  </property>
```

Comment:
I don't think this should be exposed to users to configure.

Reply:
i see. Let me put this value as private attribute in WritableRatisContainerProvider only.